go.mod

```diff
@@ -1,100 +1,83 @@
 module github.com/openshift/rbac-permissions-operator
 
-go 1.17
+go 1.19
 
 require (
 	github.com/golang/mock v1.6.0
 	github.com/onsi/ginkgo v1.16.5
-	github.com/onsi/gomega v1.20.1
+	github.com/onsi/gomega v1.24.1
```

Comment:
I didn't look at updating any of RPO's other dependencies other than OCP/K8s specifically. But I have bumped gomega's dependency now so that it aligns with your MR too.